content/pt-br/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -0,0 +1,400 @@
+kubectl config --kubeconfig=config-demo set-cluster test --server=https://5.6.7.8 --insecure-skip-tls-verify
+```
+
+Adicione detalhes do usuário ao seu arquivo de configuração:
+
+{{< caution >}}
+O salvamento de senhas na configuração do cliente do Kubernetes é arriscado. Uma alternativa melhor seria usar um plug-in de credenciais e salvá-las separadamente. Veja: [client-go credential plugins](/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins)
+{{< /caution >}}
+
+```shell
+kubectl config --kubeconfig=config-demo set-credentials developer --client-certificate=fake-cert-file --client-key=fake-key-seefile
+kubectl config --kubeconfig=config-demo set-credentials experimenter --username=exp --password=some-password
+```
+
+{{< note >}}
+- Para excluir um usuário, é possível executar `kubectl --kubeconfig=config-demo config unset users.<name>`
+- Para remover um cluster, é possível executar `kubectl --kubeconfig=config-demo config unset clusters.<name>`
+- Para remover um contexto,  é possível executar `kubectl --kubeconfig=config-demo config unset contexts.<name>`
+{{< /note >}}
+
+Adicione detalhes de contexto ao seu arquivo de configuração:
+
+```shell
+kubectl config --kubeconfig=config-demo set-context dev-frontend --cluster=development --namespace=frontend --user=developer
+kubectl config --kubeconfig=config-demo set-context dev-storage --cluster=development --namespace=storage --user=developer
+kubectl config --kubeconfig=config-demo set-context exp-test --cluster=test --namespace=default --user=experimenter
+```
+
+Abra seu arquivo `config-demo` para ver os detalhes adicionados. Como alternativa para abrir o arquivo `config-demo`, você pode usar o comando `config view`
+
+```shell
+kubectl config --kubeconfig=config-demo view
+```
+
+O resultado mostra os dois clusters, dois usuários e três contextos:
+
+```yaml
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: fake-ca-file
+    server: https://1.2.3.4
+  name: development
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://5.6.7.8
+  name: test
+contexts:
+- context:
+    cluster: development
+    namespace: frontend
+    user: developer
+  name: dev-frontend
+- context:
+    cluster: development
+    namespace: storage
+    user: developer
+  name: dev-storage
+- context:
+    cluster: test
+    namespace: default
+    user: experimenter
+  name: exp-test
+current-context: ""
+kind: Config
+preferences: {}
+users:
+- name: developer
+  user:
+    client-certificate: fake-cert-file
+    client-key: fake-key-file
+- name: experimenter
+  user:
+    # Documentation note (this comment is NOT part of the command output).
+    # Storing passwords in Kubernetes client config is risky.
+    # A better alternative would be to use a credential plugin
+    # and store the credentials separately.
+    # See https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins
```

Comment:
What do you think of translating the comment here? It provides important context about what's going on.